src/DynamoCoreWpf/ViewModels/PackageManager/PackagePathViewModel.cs

```diff
@@ -132,8 +172,9 @@ private void RaiseCanExecuteChanged()
         private bool CanDelete(int param)
         {
             var programDataPackagePathIndex = GetIndexOfProgramDataPackagePath();
+            var appDataPackagePathIndex = GetIndexOfDefaultAppDataPackagePath();
             if (RootLocations.IndexOf(Resources.PackagePathViewModel_BuiltInPackages) == param ||
-                    programDataPackagePathIndex == param)
+                    programDataPackagePathIndex == param || appDataPackagePathIndex == param)
```

Comment:
yeah, while it's funny that nothing can be deleted now, it also makes sense...

Reply:
@sm6srw why are you preventing appdata from being deleted?

Reply:
Is it because it's the default download path?

Reply:
I think - yes - because if a user removes it, it comes right back on restart.

Reply:
Yes, this is the path that we always add at startup if missing. A lot of strange edges cases never materializes if we prevent this path from being removed/edited. I can't see any value in the previous behavior. It is just confusing.